tests/run-make/unconditional-drop-recursion/main.rs

```diff
@@ -0,0 +1,37 @@
+//! This can not be a regular ui test since we can only lint drop recursion
+//! after the drop elaboration MIR pass, which is not performed in ui tests.
```

Comment:
Can't you use the `build-error` directive to achieve the same goal with UI tests ?
```suggestion
// build-error
```

Reply:
I think you're right, thanks! Will fix. (Assuming you mean [`build-fail`](https://rustc-dev-guide.rust-lang.org/tests/ui.html#controlling-passfail-expectations))